test/unit/api/test_api.py

```diff
@@ -97,6 +97,37 @@ def test_get_file_info(self):
             assert isinstance(result, VFileVersion)
             assert result == created_file
 
+    def test_get_file_info_by_name(self):
```

Comment:
There is no test for hidden file, for non-existent file, and for file with replication/SSE/legal hold.

Reply:
ok, will write some more tests, once we are done with the basic review

Reply:
@agoncharov-reef added new tests